public/icons/zoom.svg

```diff
@@ -0,0 +1 @@
+<svg baseProfile="tiny" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M32 28.4l-8.5-8.5c3.3-5 2.8-11.7-1.6-16.1-5-5-13.1-5-18.1 0-5 5-5 13.1 0 18.1 4.4 4.4 11.2 4.9 16.1 1.6l8.5 8.5 3.6-3.6zm-24.6-10.1c-3-3-3-7.9 0-10.9s7.9-3 10.9 0 3 7.9 0 10.9c-3.1 3-7.9 3-10.9 0z"/></svg>
```

Comment:
mogłabyś wygenerować z tych wszystkich svg icon font, np. https://icomoon.io/app/#/select
taka nadmiarowość powoduje, że aplikacja bardzo długo się wczytuje (duża ilość assetów) w stosunku do jednego icon'fonta. Ma to znaczenie szczególnie na mobile gdzie samo ustanowienie połączenia trwa czasami dłużej niż faktyczne pobieranie